Reduce the log level of the message sent when disconnected  from sentinel discovery service

In case of scheduled maintenance of Redis Enterprise and Sentinel Discovery Service is temporary down, Jedis Client reports ERROR level messages which    triggers internal alerts and causes too much noise in some Production deployments
Could we downgrade this particular message  to a lower lever  ?

@ggivo  Hi

below is the full stack trace:
I am trying get more details from the client and will share
Thanks

```
Caused by: fr.ca.cat.sdj.common.storage.exception.StorageTechnicalErrorException: redis.clients.jedis.exceptions.JedisConnectionException: Failed to connect to any host resolved for DNS name."," fr.ca.cat.sdj.common.storage.module.legacy.DataStore.store(DataStore.java:118)","time=2024-03-25T12:47:33.485+0100|level=ERROR|env=u1|event_cod=|event_typ=|app_id=ress-PartageContexte|app_version=v1.4.0|component_id=redis.clients.jedis.JedisSentinelPool|thread_id=|req_id=|uom_cod=|corr_id=|usr_id=|message=Lost connection to Sentinel at 10.77.213.66:8001. Sleeping 5000ms and retrying.|exception= redis.clients.jedis.exceptions.JedisConnectionException: Failed to create socket.
 at redis.clients.jedis.DefaultJedisSocketFactory.createSocket(DefaultJedisSocketFactory.java:110)
 at redis.clients.jedis.Connection.connect(Connection.java:226)
 at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:144)
 at redis.clients.jedis.BinaryJedis.connect(BinaryJedis.java:314)
 at redis.clients.jedis.BinaryJedis.initializeFromClientConfig(BinaryJedis.java:92)
 at redis.clients.jedis.BinaryJedis.(http://binaryjedis.java:87/)
 at redis.clients.jedis.Jedis.(http://jedis.java:56/)
 at redis.clients.jedis.JedisSentinelPool$MasterListener.run(JedisSentinelPool.java:391)
Caused by: java.net.ConnectException: Connection refused
 at java.base/sun.nio.ch.Net.pollConnect(Native Method)
 at java.base/sun.nio.ch.Net.pollConnectNow(Unknown Source)
 at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(Unknown Source)
 at java.base/sun.nio.ch.NioSocketImpl.connect(Unknown Source)
 at java.base/java.net.SocksSocketImpl.connect(Unknown Source)
 at java.base/java.net.Socket.connect(Unknown Source)
 at redis.clients.jedis.DefaultJedisSocketFactory.createSocket(DefaultJedisSocketFactory.java:80)
 ... 7 more
```

--- a/src/main/java/redis/clients/jedis/JedisSentinelPool.java
+++ b/src/main/java/redis/clients/jedis/JedisSentinelPool.java
@@ -407,8 +407,8 @@ public class JedisSentinelPool extends Pool<Jedis> {
         } catch (JedisException e) {
 
           if (running.get()) {
-            LOG.warn("Lost connection to Sentinel at {}:{}. Sleeping 5000ms and retrying.", host,
-              port, e);
+            LOG.warn("Lost connection to Sentinel {}:{}. Sleeping {}ms and retrying.", host, port, subscribeRetryWaitTimeMillis,
+                    e);
             try {
               Thread.sleep(subscribeRetryWaitTimeMillis);
             } catch (InterruptedException e1) {

--- a/src/main/java/redis/clients/jedis/providers/SentineledConnectionProvider.java
+++ b/src/main/java/redis/clients/jedis/providers/SentineledConnectionProvider.java
@@ -310,7 +310,7 @@ public class SentineledConnectionProvider implements ConnectionProvider {
         } catch (JedisException e) {
 
           if (running.get()) {
-            LOG.error("Lost connection to sentinel {}. Sleeping {}ms and retrying.", node,
+            LOG.error("Lost connection to Sentinel {}. Sleeping {}ms and retrying.", node,
                 subscribeRetryWaitTimeMillis, e);
             try {
               Thread.sleep(subscribeRetryWaitTimeMillis);

